packages/ember-glimmer/lib/helpers/mut.js

```diff
@@ -47,8 +45,13 @@ import { INVOKE } from './action';
 
   The `mut` helper changes the `totalClicks` value to what was provided as the action argument.
 
-  See a [2.0 blog post](http://emberjs.com/blog/2015/05/10/run-up-to-two-oh.html#toc_the-code-mut-code-helper) for
-  additional information on using `{{mut}}`.
+  ### Legacy behaviour
```

Comment:
Can we avoid this whole section? Do we have to document what is in `this.attrs`?